### DISCUSSION TITLE
Can you provide an example for trigger.dev connect?

### DISCUSSION THREAD

**Discussion by danbars** [triggerdotdev/trigger.dev#717] [Q&A] [answered]
Is [trigger.dev connect](https://trigger.dev/docs/documentation/concepts/connect) available already? 
Can you please provide an example?

iiuc, I should be able to use it to provide my own users' tokens for the same job, executed for a different user every time. 
For example, I'll have a job that notifies slack. The job will be triggered from my own server using [sendEvent](https://trigger.dev/docs/sdk/triggerclient/instancemethods/sendevent), and the specific user's token will be used in that run. 
Did I get this correctly?

Thanks!

**Comment by ericallam** [comment, marked as answer]
@danbars Trigger.dev Connect isn't available yet but we have built a way for you to provide your users authentication credentials during job runs, we call it [Bring Your Own Auth](https://trigger.dev/docs/documentation/guides/using-integrations-byo-auth)
